specs/deposits.md

```diff
@@ -241,19 +246,41 @@ This transaction MUST have the following values:
    contract][predeploy]).
 3. `mint` is `0`
 4. `value` is `0`
-5. `gasLimit` is set to 150,000,000.
-6. `isSystemTx` is set to `true`.
-7. `data` is an [ABI] encoded call to the [L1 attributes predeployed contract][predeploy]'s
-   `setL1BlockValues()` function with correct values associated with the corresponding L1 block (cf.
-   [reference implementation][l1-attr-ref-implem]).
-
-If the Regolith upgrade is active, some fields are overridden:
-
-1. `gasLimit` is set to 1,000,000
-2. `isSystemTx` is set to `false`
-
-This system-initiated transaction for L1 attributes is not charged any ETH for its allocated `gasLimit`,
-as it is effectively part of the state-transition processing.
+5. `gasLimit` is set to 150,000,000 prior to the Regolith upgrade, and 1,000,000 after.
+6. `isSystemTx` is set to `true` prior to the Regolith upgrade, and `false` after.
+7. `data` is an encoded call to the [L1 attributes predeployed contract][predeploy] that
+depends on the upgrades that are active (see below).
+
+This system-initiated transaction for L1 attributes is not charged any ETH for its allocated
+`gasLimit`, as it is considered part of state-transition processing.
+
+### L1 Attributes Deposited Transaction Calldata
+
+Prior to the Ecotone upgrade, the `data` field of the L1 attributes deposited transaction is an
+[ABI][ABI] encoded call to the `setL1BlockValues()` function with correct values associated with
+the corresponding L1 block (cf.  [reference implementation][l1-attr-ref-implem]).
+
+If the Ecotone upgrade is active, then `data` is instead a call to the `setL1BlockValuesEcotone()`
```

Comment:
Nit: this name feels a bit clunky. 

Changing the arguments will allow this to exist alongside the existing function, is the `Ecotone` postfix desirable?

Reply:
The thinking here was to make it clear the functions are in fact independent and post-Ecotone upgrade you must call the  new one.

Reply:
Sounds reasonable. I won't block on this, but I would be a little bit worried about having a lot of this upgrade name post-fix scar tissue kicking around though.
I'd be interested in hear thoughts from other reviewers as they drive by. :)

Reply:
The idea is that we postfix everything with the hardfork name starting with bedrock. This gives a very clear and forwards compatible way of doing things across the whole codebase. fwiw, the geth codebase follows a similar pattern